tests/OptionsTest.php

```diff
@@ -653,6 +653,37 @@ public function testReleaseOptionDefaultValueIsPreferredFromSentryEnvironmentVar
         $this->assertSame('0.0.4', (new Options())->getRelease());
     }
 
+    /**
+     * @backupGlobals enabled
+     *
+     * @dataProvider spotlightEnvironmentValueDataProvider
+     */
+    public function testSpotlightOptionDefaultValueIsControlledFromEnvironmentVariable(string $environmentVariableValue, bool $expectedSpotlightEnabled, string $expectedSpotlightUrl): void
+    {
+        $_SERVER['SENTRY_SPOTLIGHT'] = $environmentVariableValue;
+
+        $options = new Options();
+
+        $this->assertEquals($expectedSpotlightEnabled, $options->isSpotlightEnabled());
+        $this->assertEquals($expectedSpotlightUrl, $options->getSpotlightUrl());
+    }
+
+    public static function spotlightEnvironmentValueDataProvider(): array
+    {
+        $defaultSpotlightUrl = 'http://localhost:8969';
+
+        return [
+            ['', false, $defaultSpotlightUrl],
+            ['true', true, $defaultSpotlightUrl],
+            ['1', true, $defaultSpotlightUrl],
+            ['false', false, $defaultSpotlightUrl],
+            ['0', false, $defaultSpotlightUrl],
+            ['null', false, $defaultSpotlightUrl],
+            ['http://localhost:1234', true, 'http://localhost:1234'],
+            ['some invalid looking value', false, $defaultSpotlightUrl],
```

Comment:
This is the logic we used in Python SDK first (and then ported to JS): https://github.com/getsentry/sentry-python/pull/3443/files#diff-e9f88ca6368ce622839e078ed26c741a927752c01ceae64cd504d3ae86de6fd2R74-R75